war/src/main/less/abstracts/theme.less

```diff
@@ -83,26 +83,33 @@
   --alert-default-color: #0c5464;
 
   // Success
-  --alert-success-icon-color: #4caf50;
-  --alert-success-bg-color: #d4edda;
-  --alert-success-border-color: #c3e6cb;
-  --alert-success-color: var(--success);
+  --notification-success-icon-color: #4caf50;
+  --notification-success-bg-color: #d4edda;
+  --notification-success-border-color: #c3e6cb;
+  --notification-success-color: var(--success);
 
   // Warning
-  --alert-warn-icon-color: #ff9800;
-  --alert-warn-bg-color: #fff3cd;
-  --alert-warn-border-color: #ffeeba;
-  --alert-warn-color: #856404;
+  --notification-warn-icon-color: #ff9800;
+  --notification-warn-bg-color: #fff3cd;
+  --notification-warn-border-color: #ffeeba;
+  --notification-warn-color: #856404;
 
   // Error
-  --alert-err-icon-color: #f44336;
-  --alert-err-bg-color: #f8d7da;
-  --alert-err-border-color: #f5c6cb;
-  --alert-err-color: #721c24;
+  --notification-err-icon-color: #f44336;
+  --notification-err-bg-color: #f8d7da;
+  --notification-err-border-color: #f5c6cb;
+  --notification-err-color: #721c24;
```

Comment:
Nitpicking but shouldn't we use the same naming scheme as on the other ones? Either warn & err or warning & danger.

Reply:
I'll adjust these ones

Reply:
Whatever you consider best!